subgraphs/cellars/schema.graphql

```diff
@@ -23,6 +23,14 @@ type Wallet @entity {
   dayDatas: [WalletDayData!]! @derivedFrom(field: "wallet")
```

Comment:
Referencing  the deposits here would be great.

Reply:
Added in